src/main.rs

```diff
@@ -101,21 +113,36 @@ fn do_edits(dotreplit_filepath: &Path, msg: &str, return_output: bool) -> Result
         .parse::<DocumentMut>()
         .with_context(|| format!("error: parsing file - {:?}", &dotreplit_filepath))?;
 
+    let mut changed: bool = false;
+    let mut outputs: Vec<Value> = vec![];
     for op in json {
+        let path = op.path;
         match op.op {
             OpKind::Add => {
                 let value = op.value.context("error: expected value to add")?;
-                handle_add(&op.path, &value, &mut doc)?
+                changed = true;
+                handle_add(&path, &value, &mut doc)?
+            }
+            OpKind::Get => {
+                let op = traversal::TraverseOps::Get(Box::new(|val| val.to_value()));
```

Comment:
now that I'm reviewing this, I think the `FnOnce` is yagni.

The intent was to avoid presuming that we wanted to get a `serde_json::Value`, but considering the ops we're doing, I think that's a reasonable assumption. If this were to be broken out into a reusable library, or rolled up into `toml_edit` or something that would be useful, but not here.

Reply:
i think it'll be useful for `add` and `remove` :)